mrbgems/mruby-range-ext/src/range.c

```diff
@@ -162,14 +164,27 @@ mrb_range_size(mrb_state *mrb, mrb_value range)
     num_p = FALSE;
   }
   if (num_p) {
-    double f;
-
-    if (beg_f > end_f) return mrb_fixnum_value(0);
-    f = end_f - beg_f;
-    if (!r->excl) {
-      return mrb_fixnum_value((mrb_int)ceil(f + 1));
+#ifndef DBL_EPSILON
+#define DBL_EPSILON 2.2204460492503131e-16
+#endif
+    double n = end_f - beg_f;
```

Comment:
I did'n know that.
Should we use `mrb_float` instead of `double` and `float` in all cases?